topologies/atedut_3.testbed

```diff
@@ -0,0 +1,47 @@
+# proto-file: github.com/openconfig/ondatra/blob/main/proto/testbed.proto
```

Comment:
Revert this file? Those who need the 3 port variant should just use atedut_4. Also please revert the changes to file mode in the other *.binding and *.testbed files.

Please rebase to the latest main. I added some file mode checks.